bin/zenrun.d/patch.sh

```diff
@@ -27,15 +27,33 @@ applyPatches() {
         return "$RC"
     fi
 
-    # First pop everything off (the image may or may not actually have the
-    # patches applied, so need to pop them first)
-    quilt pop -a
+    # Assure that patch directory and series file exist.
+    if [[ ! -d patches ]]; then
+        mkdir patches
+    fi
+    touch patches/series
+
+    # Check for new patches
+    quilt unapplied > /dev/null 2>&1
     RC=$?
     if [[ $RC != 0 ]]; then
```

Comment:
Technically this gives us the status if the redirection, not the ```quilt unapplied``` command.  This issue exists in other places in this script as well.